qiskit/circuit/library/standard_gates/x.py

```diff
@@ -1330,6 +1336,17 @@ class MCXRecursive(MCXGate):
         2. Iten et al., 2015. https://arxiv.org/abs/1501.06911
     """
 
+    @deprecate_func(
+        additional_msg=(
+            "It is recommended to use MCXGate and let HighLevelSynthesis choose "
+            "the best synthesis method depending on the number of ancilla qubits available. "
+            "If this specific synthesis method is required, one can specify it using the "
+            "high-level-synthesis plugin `mcx_1_clean` for MCX gates, or, alternatively, "
```

Comment:
Are these names still correct after the renaming? In the tests you're using 
```suggestion
            "high-level-synthesis plugin '1_clean_b95' for MCX gates, or, alternatively, "
```